topostats/validation.py

```diff
@@ -45,6 +45,7 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
             ".asd",
             ".jpk",
             ".ibw",
+            ".gwy",
```

Comment:
The `.gwy` file format does not seem to have channels. It appears to be structured so that the first object within the initial object contains a component with the heightmap data. There isn't really a channel name associated with it aside from the component name of 'data'.